examples/ads.amp.html

```diff
@@ -440,13 +440,21 @@ <h2>AdSpirit</h2>
       data-asm-host="help.adspirit.de">
   </amp-ad>
 
-  <h2>AdStir 320x50 banner</h2>
+  <h2>adstir 320x50 banner</h2>
   <amp-ad width="320" height="50"
       type="adstir"
       data-app-id="MEDIA-343ded3e"
       data-ad-spot="1">
   </amp-ad>
 
+  <h2>adstir native</h2>
```

Comment:
The example seems to be a no fill ad when I tested it in Chrome. Is this expected? If so I would suggest either removing, or add a line `<p>Due to ad targeting, the slot might not load ad.</p>`